walkeeper/src/replication.rs

```diff
@@ -113,6 +118,16 @@ impl ReplicationConn {
                             let reply = StandbyReply::des(&m[1..])
                                 .context("failed to deserialize StandbyReply")?;
                             state.disk_consistent_lsn = reply.write_lsn;
+                            // Meaning of `flush_lsn` and `write_lsn`
+                            // is far from obvious, but it is what walreceiver returns.
```

Comment:
oh yes

Reply:
Btw, I fixed it in the first commit of #866 : [cf275011](https://github.com/zenithdb/zenith/pull/866/commits/cf275011dc7e52763b4a4bc8be04f92d13aeab2d)